onnx_tf/handlers/backend/dilated_pooling.py

```diff
@@ -613,3 +621,18 @@ def dilated_maxpool(self, force_custom_impl=False):
                                 strides=self.kernel_shape, padding="VALID",
                                 pooling_type=self.pooling_type)
         return pooled
+
+    def is_supported(self):
+        """
+            Function to check if the current set of arguments are
+            supported for average pool
+        """
+        if self.pooling_type.startswith("MAX"):
+            return True
+        else:
+            return self.count_include_pad or ((
```

Comment:
Can we break this long logical expression down a little bit for readability?

Reply:
@tjingrant thanks for the comment. I split this expression in multiple if / elif statements with comments.
Thanks!